apps/test/unit/templates/progress/progressHelpersTest.js

```diff
@@ -83,7 +83,7 @@ describe('progressHelpers', () => {
     const unlockedLevels = fakeLevels(3);
     const lockedLevels = fakeLevels(3).map((level, index) => ({
       ...level,
-      locked: index === 2 ? true : false // lock last level in level group
+      isLocked: index === 2 ? true : false // lock last level in level group
```

Comment:
`index === 2` evaluates to a boolean so I don't think you need the ternary (same comment below)

Reply:
lol i tend to not even look at the surrounding code when simply updating a name, but i touched the line so i'll clean it up. ;)